Add first headline feature (#1805)
theforeman/theforeman.org: _includes/manuals/nightly/1.2_release_notes.md

### Headline features

#### Audits listing performance tweaks

> Loading audits could get very slow and resource consuming in case the audits were scoped to specific organization or location. With this release, the SQL queries were optimized to load the correct audit records faster. ([#30053](https://projects.theforeman.org/issues/30053))

Reviewer: ```suggestion
Loading audits could be very slow and resource consuming when the audits were scoped to a specific organization or location. With this release, the SQL queries were optimized to load the correct audit records faster. ([#30053](https://projects.theforeman.org/issues/30053))
```
